package.json

```diff
@@ -40,7 +40,9 @@
     "inquirer": "^7.3.0",
     "json-source-map": "^0.6.1",
     "mocha": "^7.2.0",
-    "nock": "^12.0.3"
+    "nock": "^12.0.3",
+    "xml2json": "^0.12.0",
+    "xpath-to-css": "^1.0.5"
```

Comment:
Adding these as devDependencies as I think we consider tosback-import.js a dev task, not a production functionality.